src/tests/streams.cpp

```diff
@@ -234,11 +233,15 @@ TEST_F(rnp_tests, test_stream_file)
     /* read file back, checking the contents */
     assert_rnp_success(init_file_src(&src, filename));
     for (int i = 0; i < iterations; i++) {
-        assert_int_equal(src_read(&src, tmpbuf, filedatalen), filedatalen);
+        size_t read = 0;
+        assert_true(src_read(&src, tmpbuf, filedatalen, &read));
+        assert_int_equal(read, filedatalen);
         assert_int_equal(memcmp(tmpbuf, filedata, filedatalen), 0);
     }
     for (int i = 0; i < 5 * iterations; i++) {
-        assert_int_equal(src_read(&src, tmpbuf, 3), 3);
+        size_t read = 0;
+        assert_true(src_read(&src, tmpbuf, 3, &read));
```

Comment:
Simplify by using `src_read_eq`?

Reply:
@rrrooommmaaa I thought whether to replace it with src_read_eq(). While it doesn't make major difference, these two lines will test function src_read() and two cases (return true and read == 3), while src_read_eq() will test only this function and i'ts result.